bindings/matrix-sdk-ffi/src/timeline/mod.rs

```diff
@@ -68,7 +68,7 @@ use crate::{
     RUNTIME,
 };
 
-mod content;
+pub mod content;
```

Comment:
Is it a necessary change?

Reply:
yeah I need for  `get_loaded_reply_details`

Reply:
I need it to be able to be able to import it here:
https://github.com/matrix-org/matrix-rust-sdk/blob/4b88fe1078f6905cb9b67c8851478b035510804f/bindings/matrix-sdk-ffi/src/room.rs#L38-L40